bundles/org.openhab.binding.broadlinkthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -95,59 +103,95 @@
 		<label>Active</label>
 		<description>Shows if thermostat is currently actively heating</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Heating</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="roomtemperature">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature</label>
 		<description>Room temperature, measured directly at the device</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="roomtemperatureexternalsensor">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature Ext. Sensor</label>
 		<description>Room temperature, measured by the external sensor</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="setpoint">
 		<item-type>Number:Temperature</item-type>
 		<label>Setpoint</label>
 		<description>Temperature setpoint that open/close valve</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" step="0.5"/>
 	</channel-type>
 	<channel-type id="temperatureoffset">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature Offset</label>
 		<description>Manual temperature adjustment</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" step="0.5" min="-2.5" max="2.5"/>
 	</channel-type>
 	<channel-type id="temperature">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature</label>
 		<description>Temperature</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="remotelock">
 		<item-type>Switch</item-type>
 		<label>Remote Lock</label>
 		<description>Locks the device to only allow remote actions</description>
 		<category>Lock</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
What do you think about this one? Should this also be tagged with the new LockState ?